source/2020-09-01-ember-3-21-released.md

```diff
@@ -0,0 +1,108 @@
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.21
+
+Ember.js 3.21 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+#### New Features
+
+Ember.js 3.21 introduced no new features.
+
+#### Deprecations
+
+Ember.js 3.21 introduced 1 deprecation.
+
+1. `getWithDefault` (both the function and class method) has been [deprecated](https://github.com/emberjs/ember.js/pull/18993) per [RFC #554](https://github.com/emberjs/rfcs/blob/master/text/0554-deprecate-getwithdefault.md).
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.21, please review the [Ember.js 3.21.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.21.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.21
+
+Ember Data 3.21 release saw a few improvements to documentation and continuous integration.
+
+#### New Features
+
+Ember Data 3.21 introduced no new features.
+
+#### Deprecations
+
+Ember Data 3.21 introduced no deprecations.
+
+For more details on changes in Ember Data 3.21, please review the
+[Ember Data 3.21.0 release page](https://github.com/emberjs/data/releases/tag/v3.21.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+<!--alex ignore easy-->
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npx ember-cli-update
+```
+
+This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
```

Comment:
```suggestion
This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the GitHub README](https://github.com/ember-cli/ember-cli-update).
```

Reply:
Thanks! I think the casing came from copy-paste. I fixed it in a commit.